api/lib/domain/events/handle-certification-rescoring.js

```diff
@@ -340,21 +349,30 @@ function _createV2AssessmentResult({
       assessmentId: certificationAssessment.id,
       juryId: event.juryId,
     });
-  } else if (certificationAssessmentScore.hasInsufficientCorrectAnswers()) {
-    return AssessmentResultFactory.buildInsufficientCorrectAnswers({
-      emitter,
+  } else if (!certificationAssessmentScore.hasEnoughNonNeutralizedChallengesToBeTrusted) {
+    return AssessmentResultFactory.buildNotTrustableAssessmentResult({
       pixScore: certificationAssessmentScore.nbPix,
       reproducibilityRate: certificationAssessmentScore.getPercentageCorrectAnswers(),
+      status: certificationAssessmentScore.status,
       assessmentId: certificationAssessment.id,
+      emitter,
       juryId: event.juryId,
     });
-  } else if (!certificationAssessmentScore.hasEnoughNonNeutralizedChallengesToBeTrusted) {
-    return AssessmentResultFactory.buildNotTrustableAssessmentResult({
+  } else if (_isLackOfAnswersForTechnicalReason({ certificationAssessmentScore, certificationCourse })) {
+    return AssessmentResultFactory.buildLackOfAnswersForTechnicalReason({
```

Comment:
je re challenge notre choix d'avoir repris cet assessment result à cause du terme `lack of answers` qui n'est pas correct vis à vis de notre cas.

Ce n'est pas annulé suite à un manque de réponse au global mais bien d'un manque de bonne réponse.

Reply:
On change quelque chose @AndreiaPena ?
On introduit un nouveau builder avec un nom plus spécifique ?

Reply:
Oui je préférerais, à voir si on fusionne pas tout ça dans le refacto du scoring/rescoring après